tokenomics/etherdai.md

```diff
@@ -0,0 +1,33 @@
+# EtherDai
+
+EtherDai is a synthetic Ether token backed by liquid staked Ether tokens and debt issued by Maker. It is pegged to the value of Ether. EtherDai is one of the central products of the Endgame Plan. 
+
+{% hint style="warning" %}
+This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date.
+{% endhint %}
+
+## Endgame Interactions
+
+Maker Core onboards EtherDai as one or more Maker Vaults, allowing users to generate DAI using EtherDAI as collateral. Owners of EtherDAI vaults receive significant SubDAO token emissions through a yield farming system.
+
+All SubDAOs earn a revenue share of the fees generated by EtherDAI vaults.
+
+EtherDai is a component of the Elixir liquidity pool token.
+
+The Protocol Owned Vault accumulates EtherDAI using generated Dai. This slightly increases the ratio of decentralized collateral backing Dai.
+
+Responsibility for EtherDai comes under the Protocol Engineering Scope.
+
+## EtherDai Development
+
+Initially, Maker will create EtherDai as a simple ERC20 token wrapper around Lido stETH. This will reduce the time required for implementation, allowing Maker to release EtherDai as soon as possible.
+
+Later, Maker will be able to upgrade the EtherDai implementation, converting it into a full synthetic asset. This will require a new deployment of the Multi-Collateral Dai smart contracts, with EtherDai taking the place of Dai.
+
+## Related Pages
```

Comment:
Just checking if we're still keeping the Related Pages section. If not, to be removed